tests/Hyperwallet/Tests/HyperwalletTest.php

```diff
@@ -3966,4 +4182,415 @@ public function testListVenmoAccountStatusTransitions_withParameters()
         \Phake::verify($apiClientMock)->doGet('/rest/v3/users/{user-token}/venmo-accounts/{venmo-account-token}/status-transitions', array('user-token' => 'test-user-token', 'venmo-account-token' => 'test-venmo-account-token'), array('test' => 'value'));
     }
 
+    public function testCreateTransferRefund_noClientRefundId(){
+
+        $userName = "test-username";
+        $password = "test-password";
+        $sourceAmount = 20.0;
+        $notes = "notes";
+        $memo = "memo";
+
+        $transferRefund = new TransferRefund();
+        $transferRefund->setSourceAmount($sourceAmount);
+        $transferRefund->setNotes($notes);
+        $transferRefund->setMemo($memo);
+        $transferToken = "transferToken";
+
+        $client = new Hyperwallet($userName, $password);
+
+        try {
+            $client->createTransferRefund($transferToken, $transferRefund);
+            $this->fail('HyperwalletArgumentException expected');
+        } catch (HyperwalletArgumentException $e) {
+            $this->assertEquals('clientRefundId is required!', $e->getMessage());
+        }
+
+
+    }
+
+    public function testCreateTransferRefund_noTransferRefund(){
+
+        $userName = "test-username";
+        $password = "test-password";
+        $transferToken = "transferToken";
+            $client = new Hyperwallet($userName, $password);
+
+        try {
+            $client->createTransferRefund($transferToken, null);
+            $this->fail('HyperwalletArgumentException expected');
+        } catch (HyperwalletArgumentException $e) {
+            $this->assertEquals('transferRefund is required!', $e->getMessage());
+        }
+    }
+
+
+    public function testCreateTransferRefund_noTransferToken(){
+
+        $userName = "test-username";
+        $password = "test-password";
+        $clientRefundId = 6712348070812;
```

Comment:
clientRefundId is a string. Pls change it.

Reply:
Done